lib/rubocop/cop/lint/number_convertion.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for usage of #to_i.
```

Comment:
The cop should also cover `to_f` and `Float`. Maybe some of the other numeric conversions as well (e.g `to_c`).